utils/gen-styleguide-prod-config.js

```diff
@@ -215,13 +208,27 @@ for (const section of styleguideConfig.sections) {
             `Design Specification: **${info.design}**\n`,
             currentDescription,
         ];
+        // If this is a prototype, then we don't want this in the prod
+        // documentation.
+        if (info.prototype) {
+            section.private = true;
+        }
         section.description = lines.filter((l) => !!l).join("  \n");
     } else {
         // eslint-disable-next-line no-console
         console.log(`No package information found for ${section.name}`);
     }
 }
 
+// See https://www.netlify.com/docs/continuous-deployment/#build-environment-variables
+if (process.env.PULL_REQUEST !== "true") {
+    // If we're not a pull request, then let's iterate over the configuration
+    // and remove private sections.
+    styleguideConfig.sections = removePrivateSections(
+        styleguideConfig.sections,
+    );
+}
```

Comment:
Same as before, but now we do it after augmenting root level sections with package info, so we can strip things we auto-marked as private.